apiserver/facades/agent/instancemutater/interface.go

```diff
@@ -16,46 +16,15 @@ import (
 type InstanceMutaterState interface {
 	state.EntityFinder
 
-	WatchModelMachines() state.StringsWatcher
```

Comment:
Removed: these were left over from when state was used instead of the model cache.